### PR TITLE
Keras QAT Docs update for standalone batchnorms

### LINKED ISSUE #3331: Add document update for keras QAT for BN

### PR DESCRIPTION
**Pull request by quic-ristha**
Fixes #3331

### CODE REVIEW

**Comment by quic-kyunggeu** [comment]
Merged in [5440457](https://github.com/quic/aimet/commit/54404573f9f01114cdc835bb5422b429cfe1d433)
